feature-libs/order/components/order-history/myaccount-v2/consolidated-information/myaccount-v2-order-consolidated-information.component.ts

```diff
@@ -0,0 +1,114 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {
+  Component,
+  ChangeDetectionStrategy,
+  Input,
+  inject,
+} from '@angular/core';
+import { OrderEntry } from '@spartacus/cart/base/root';
+import { Images } from '@spartacus/core';
+import { OrderConsignmentsService } from '../../../order-details';
+import {
+  ConsignmentView,
+  OrderView,
+  OrderHistoryView,
+} from '@spartacus/order/root';
+import { OrderCriticalStatus } from '../myaccount-v2-order-history.model';
+
+@Component({
+  selector: 'cx-myaccount-v2-order-consolidated-information',
+  templateUrl: './myaccount-v2-order-consolidated-information.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class MyAccountV2OrderConsolidatedInformationComponent {
+  protected orderConsignmentsService = inject(OrderConsignmentsService);
+  protected criticalStatuses = Object.values(OrderCriticalStatus);
+  @Input()
+  order?: OrderHistoryView;
+  protected IMAGE_COUNT = 4; //showing fixed no.of images, without using carousel
+  getConsignmentsCount(consignments: ConsignmentView[] | undefined): number {
+    let count = 0;
+    if (consignments) {
+      for (const consignment of consignments) {
+        if (consignment.entries) {
+          count = count + consignment.entries.length;
+        }
+      }
+    }
+    return count;
+  }
+
+  getOrderEntriesCount(orderEntries: OrderEntry[] | undefined): number {
+    if (orderEntries) {
+      return orderEntries.length;
+    }
+    return 0;
+  }
+
+  isStatusCritical(status: string): boolean {
+    if (
+      this.criticalStatuses.includes(
+        status.toUpperCase() as OrderCriticalStatus
+      )
+    ) {
+      return true;
+    } else {
+      return false;
+    }
```

Comment:
[NITPICK]
What about simply:
```suggestion
    return this.criticalStatuses.includes(
      status.toUpperCase() as OrderCriticalStatus
    )
```

Reply:
done